### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Microsoft IIS Home Page_Keyboard: keyboard focus is not moving to "options" menu button and hence it is not accessible through keyboard.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#57]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106e

**MAS ID Violated**: MAS2.4.3, 2.1.1

**Repro Steps:**
1. Open "Microsoft IIS" Home Page using the URL: https://manage.iis.net/
2. Tab till "Not Connected" link.
3. Press Tab and validate that focus is moving to "options" button.

**Actual Result:**
keyboard focus is not moving to "options" menu button and hence it is not accessible through keyboard.

**Expected Result:**
When user press Tab Key after "Options" button, keyboard focus should move to "Options" menu button and user must be able to expand it and access all its options using keyboard.

**User Impact:**
Keyboard only users will be unable to access the options under "options" menu button.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment:**
[Bug 57_Option menu button is not accessible using keyboard.pptx](https://github.com/Microsoft/IIS.WebManager/files/2317146/Bug.57_Option.menu.button.is.not.accessible.using.keyboard.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
